src/Button/CopyButton/CopyButton.tsx

```diff
@@ -0,0 +1,97 @@
+import * as React from 'react';
+import { useEffect, useState } from 'react';
```

Comment:
The `react` imports can be combined.

Reply:
i think the `import *` cannot be combined.

Reply:
See 323de2fd2b95c17038b5c0b364acbf407769c204.